am/templates/api/api-configmap.yaml

```diff
@@ -144,6 +144,50 @@ data:
         {{- end }}
       {{- end }}
       
+  {{- if .Values.api.services }}
+    {{- if .Values.api.services.notifier }}
+      notifier:
+{{ toYaml .Values.api.services.notifier | indent 8 }}
+    {{- end }}
+    {{- if .Values.api.services.certificate }}
+      certificate:
+        enabled: {{ .Values.api.services.certificate.enabled | default true}}
+        {{- if .Values.api.services.certificate.enabled }}
+        cronExpression: {{ .Values.api.services.certificate.cronExpression | default "0 0 5 * * *" }}
+        expiryThresholds: {{ .Values.api.services.certificate.expiryThresholds | default "20,15,10,5,1" }}
+          {{- if .Values.api.services.certificate.expiryEmailSubject }}
+        expiryEmailSubject: {{ .Values.api.services.certificate.expiryEmailSubject | quote }}
+          {{- end }}
+        {{- end }}
+    {{- end }}
+  {{- end }}
+
+    notifiers:
+      email: 
+        enabled: {{ .Values.api.notifiers.email.enabled }}
```

Comment:
@leleueri What is the reason of not reusing the smtp.* values and use dedicated one? Do we think that they can (really) use an other SMTP for the two purposes?

Reply:
@brasseld not it will be the same value for sure but I did it to be align with the APIM way to configure notifiers.

Reply:
Yes, I was assuming that it was for aligning it with APIM.... but... it's currently a pain to manage SMTP on APIM side because of that.
I would consider to provide ability to override the smtp for the notifiers, but if there is no override, reuse the one defined for SMTP.
wdyt ?

Reply:
:+1:

Reply:
Indeed, I would be interested to see how you leverage that... Facing same issue with other products ;)

Reply:
I use notifier values in priority and fallback to smtp settings if missing.
For truststore, this is a dedicated entry into the notifier section.